.github/workflows/nightly-release.yml

```diff
@@ -0,0 +1,106 @@
+# **what?**
+# Nightly releases to GitHub and PyPI. This workflow produce the following outcome:
+# - generate and validate data for night release (commit SHA, version number, release branch);
+# - pass data to release workflow;
+# - night build will be pushed to test PyPI;
+# - night release will be pushed to GitHub as a draft release;
+#
+# **why?**
+# Ensure an automated and tested release process for nightly builds
+#
+# **when?**
+# This workflow runs on schedule or can be run manually on demand.
+
+name: Nightly Release to GitHub and PyPi
```

Comment:
```suggestion
name: Nightly Test Release to GitHub and PyPi
```

Reply:
Fixed in https://github.com/dbt-labs/dbt-core/pull/6602/commits/2fbde2cc196d0f942da42cb03b54b25be4e95e53